ext/fetch/22_body.js

```diff
@@ -202,7 +249,7 @@
          * @returns {ReadableStream<Uint8Array> | null}
          */
         get() {
-          webidl.assertBranded(this, prototype);
+          webidl.assertBranded(this, class_.prototype);
```

Comment:
This has possibility for pollution. Instead of `class_` can you rename back to prototype, and have it be the real prototype? I don't see where we use `class_` without `.prototype`.